packages/kit/src/core/sync/create_manifest_data/index.js

```diff
@@ -14,18 +14,28 @@ import { parse_route_id } from '../../../utils/routing.js';
  *   rest: boolean;
  *   type: string | null;
  * }} Part
+ *
  * @typedef {{
```

Comment:
Jsdoc comments appear in all types when merged.
```suggestion
 * }} Part
 */
 
 /**
 * @typedef {{
```